earthpy/tests/test_extent_to_json.py

```diff
@@ -0,0 +1,56 @@
+""" Tests for the extent_to_json function """
+
+
+import pandas as pd
+import pytest
+import geopandas as gpd
+from shapely.geometry import Polygon, Point
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def list_out():
+
```

Comment:
Maybe a comment here to explain and fill this blank line?
```suggestion
    """ A JSON extent on the unit square spanning (0, 0), (1, 1). """
```